pkg/api/controllers/host.go

```diff
@@ -41,8 +41,15 @@ func (p *HostPortal) ListHosts() {
 	if !policy.Authorize(p.Ctx, "host:list") {
 		return
 	}
-	// TODO: handle query parameters
-	hosts, err := db.C.ListHosts(c.GetContext(p.Ctx))
+
+	m, err := p.GetParameters()
+	if err != nil {
+		errMsg := fmt.Sprintf("list hosts failed: %s", err.Error())
```

Comment:
This message should be changed. We are getting the parameters

Reply:
done